### ISSUE TITLE
Nowa wtyczka mb filmy.

### ISSUE THREAD

**Issue by mbebe** [mbebe/blomqvist#259]
we wtyczce:
- vizjer
- rodzinne
- znet
- watchmovies
- ninja (z tv)
[plugin.video.mbfilm-1.0.zip](https://github.com/mbebe/blomqvist/files/4325399/plugin.video.mbfilm-1.0.zip)

**Comment by Huero23** [comment]
@mbebe Dziękujemy! 👍

**Comment by rafalmar1981** [comment]
Dziękujemy

**Comment by FXcodi** [comment]
Coś nie chce działać u mnie filmninja.ws #filmy/seriale
Tv ładuje dobrze.

**Comment by sky0002** [comment]
Dalej Py 2.25, a już myślałem że wiosna dla kodi 19 . Ale i tak szacun. Prawdziwe kombo, prawie jak szampon 3 w jednym ;) Dziękuję za to co robisz.

**Comment by notoco** [comment]
bez sensu robić jeszcze dla k19. to wciąż wersja Alpha. może się wiele zmienić. i wtyczka dla k19 nie będzie działać na k18.

**Comment by mbebe** [comment]
@sky0002 
[plugin.video.mbfilm-PY3.zip](https://github.com/mbebe/blomqvist/files/4329702/plugin.video.mbfilm-PY3.zip)
wersja dzialajaca w kodi19 (build z dzis)
wymaga tez resolveurl z py3

**Comment by rafalmar1981** [comment]
no to teraz będą śmiechy....ja jade na 17.6 czy lepiej wtyczki będą działały na nowszych wersjach jeśli tak to którą wybrać.  Czy filman.cc chodzi mi o wtyczkę bo strona działa. Pozdrawiam

**Comment by notoco** [comment]
Jak wyjdzie k19 to wątpię żeby komuś się chciało na dłuższą metę prowadzić odrębne repozytoria na wtyczki dla k19 i k18, a co dopiero zastanawiać się nad kompatybilnością z k17. Może czas na aktualizację?

**Comment by rafalmar1981** [comment]
Dziękuję za szybką odpowiedź czyli pozostaję przy 17.6, czy zainstalowanie z repo mbebe programu stremlink proxy daje jakąś płynność w oglądaniu np. hejo czy mtvguide bo mam taką masakrę z zacinaniem.Neta z upc Nazwa usługi
Fiber Power 150 Mb/sPrędkość pobierania: 153.6 MbpsPrędkość wysyłania: 10.24 Mbps

**Comment by notoco** [comment]
Nie, nic nie pomoże - to zależy od serwerów zewnętrznych (hejo) lub od listy (mtvguide).
Dlaczego nie przechodzisz na k18?
BTW - Net z UPC - a jak podłączony i do czego? Jak jakikolwiek box + wifi to zawsze będzie słabo.

**Comment by rafalmar1981** [comment]
z dekodera horizon kablem, a na jaką wersję kodi 18 przejść najlepiej. Kiedyś kombinowałem z 18,6 chyba ale niema tam skórki confuense na innych się pogubiłem i zostałem na 17,6, a jaka jest różnica między obiema wersjami jeszcze propo stremlink proxy, w sportowa w ustawieniach jest opcja do wyboru bardzo dużo sportu oglądam

**Comment by notoco** [comment]
Najlepiej przejść na ostatnią aktualną. Na dziś to 18.6.  A skórka confluence jest do zainstalowania z oficjalnego repo kodi.

**Comment by rafalmar1981** [comment]
niechcę cię tu zamęczać głupimi pytaniami ale  może inaczej jak zainstaluje kodi 18.6 wejdę w dodatki repozytoria to tam ją znajdę czy coś pokręciłem

**Comment by rafalmar1981** [comment]
którą wersję kodi pobrać na androida, ARMV7A (32BIT), ARMV8A (64BIT)

**Comment by notoco** [comment]
Najlepiej ze sklepu Play. Jeśli nie, to chyba wersję 32 bit

**Comment by rafalmar1981** [comment]
oki dziękuję spokojnego weekendu

**Comment by rafalmar1981** [comment]
ale lipa niemoge sie zalogowac do zalukaj i hejo blad dodatku na 18.6 na 3filmy mozna sie zalogowac, brak obrazu na kazdym filmie. Nowy system win 7 wszystko wgrane aaaaaaaaaaaaaaaa..................masakra dobrze że nie zrobiłem tego na tv smart

**Comment by notoco** [comment]
Za mało szczegółów. Jaki błąd. Pewnie instalowałeś bez wywalenia starej wersji i się profile gryzą.

**Comment by BlackGlassSkin** [comment]
@rafalmar1981 ,masz problem z akceleracją.... chyba to znajdziesz w Metodach Renderowania

windows 7 nie jest nowy co najwyżej to świeżo postawiony

**Comment by rafalmar1981** [comment]
dzięki mistrzowie ale na pierwsze to nowy system, innego kodi nie miałem. problem z akceleracją tak pomogło kiedyś na win 7 w kodi 17,6 jak przestawiłem na metodę Renderowania, ale już na leili nic nie pomogło żadne kombinacje w Renderowaniu nie pomogło....i tyle

**Comment by BlackGlassSkin** [comment]
https://winclub.pl/forum/220-modyfikacje-srom-otnik/

https://winclub.pl/forum/218-modyfikacje-srom-otnik/

**Comment by sky0002** [comment]
@notoco Nie gniewaj się bo się znamy, ale jak by wszyscy myśleli jak Ty. To świat stał by w miejscu ;)
Pozdrawiam :)

**Comment by notoco** [comment]
Nie gniewam się. Ale nie zaleceń zwykłemu użytkownikowi wersji niestabilnej, choćby nawet i działała jak skała. Bo potem czy tu, czy na KWP jest płacz. Wersje Alpha czy Beta są dla naprawdę zaawansowanych użytkowników i developerów.

**Comment by rafalmar1981** [comment]
wprowadziłem zgrzyt przepraszam, gdzieś czytałem że mój problem może leżeć w nazwie komputera admina czy coś takiego chodziło generalnie aby nie było polskich znaków. Może to być powód?

**Comment by notoco** [comment]
Tak. To główny powód. Kodi i wiele innych programów nie lubi polskich znaków w nazwie użytkownika. Poszukaj w google jak to zmienić

**Comment by rafalmar1981** [comment]
wszystko pozmieniane teraz odinstalować tą leile i na nowo wgrać? Z ciekawości nie mamy spania?

**Comment by notoco** [comment]
Nie mam windowsa żeby podpowiedzieć, ale chyba nie trzeba przeinstalowywać. 
Od 20 lat wstaje o tej samej porze.

**Comment by rafalmar1981** [comment]
no to z wstawaniem masz reguralność powiedz mi jeszcze czy ten filman.cc to działa i zaproś mnie do znajomych czy coś na kwp by tu nie robić u mbebe niepotrzebnego młynku. Jeszcze raz sprawdziłem i mam jakieś braki w uaktualnieniach windows update jakies bledy mi wyskakuja. Naprawie i dam znać co z leilą

**Comment by Bogdan16** [comment]
witam

Na rodzinne brak materialu do wyswietlania

Pozdrawiam

**Comment by mbebe** [comment]
Na czym? Na filmach, serialach, gatunkach?  Na wszystkim?

**Comment by Bogdan16** [comment]
witam

Na filmninja.ws  UWAGA brak materialow do wyswietlenia       Na  Wszystkim

Na vizjer.pl   Na Filmy na duzo filma sie kreci i nie zalancza 

Na ekino.pw   Link nie dziala  sprobuj inny     Na Wszystkim

Na cda-filmy.online  Link nie dziala  sprobuj inny     Na Wszystkim

Pozdrawiam

**Comment by BlackGlassSkin** [comment]
> Jak wyjdzie k19 to wątpię żeby komuś się chciało na dłuższą metę prowadzić odrębne repozytoria na wtyczki dla k19 i k18, a co dopiero zastanawiać się nad kompatybilnością z k17. Może czas na aktualizację? A która wersję wybrać? Stabilną czyli k18. K19 nie nadaje się jeszcze do użytku.

**jedno repo wystarczy tylko opcje w kodi trzeba zaznaczyc/odnaczyć pokaż/ukryj niekompatybilne dodatki.**

**Comment by BlackGlassSkin** [comment]
> @sky0002
> [plugin.video.mbfilm-PY3.zip](https://github.com/mbebe/blomqvist/files/4329702/plugin.video.mbfilm-PY3.zip)
> wersja dzialajaca w kodi19 (build z dzis)
> wymaga tez resolveurl z py3
> przy zmianie na 2.25.0 dziala tez w kodi 18

na wersji z 15 marca nie działa , 
na filmach udało mi sie to troche ominąć 
linijka 348 w main.py
**except Exception,e:** zamiana na 
**except Exception:**

oraz linijka 14 
**import urlparse** wyłączenie

**#import urlparse**

jakoś kuleje cos tam zadziała seriale natomiast nie działają w ogóle

https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings

**Comment by zarluk** [comment]
Czy pojawi się Horizon Go w  twoim repo, NC go  działa świetnie i do szczęścia brakuje Horizona.

**Comment by mbebe** [comment]
horizongo dziala tylko na androidzie

**Comment by zarluk** [comment]
NC go działa też na androidzie,  a jakoś zastało zadaaptowane. Działa też przez stronę i myślałem  że analogiicznie  Horizon apka na na android i strona www zę da  się może to przenieść do kodi. Dzięki za odpowiedź.

**Comment by kashkash18** [comment]
Ale Horizon nie działa na Kodi w PC . Tylko na Androidzie

**Comment by gs188** [comment]
Wtyczka wyglada ze juz jest dosc 'polamana':
- cdafilmy: nie pokazuje nic nowego, ta sama lista od kilku dni
- ekino: lista nowa ale nic nie gra (po chwili wraca do listy)
- vizjer: to samo co cdafilmy, lista bez zmian
- rodzinnekino: to samo co vizjer i cdafilmy
- znetlive - chyba juz lezy
- filmninja - lezy
